notes/coredns-1.6.3.md

```diff
@@ -14,14 +14,20 @@ In this release we have moved the *federation* plugin to
 [github.com/coredns/federation](https://github.com/coredns/federation), but it is still fully
 functional in this version. In version 1.7.0 we expect to fully deprecate it.
 
-Further more a slew a spelling corrections and other minor improvements and polish. **And** three(!)
-new plugins.
+Further more a slew a spelling corrections and other minor improvements and polish. **And** two new
+plugins: *clouddns* and *sign*.
 
 # Plugins
 
-* [*acl*](/plugins/acl) blocks queries depending on their source IP address.
 * [*clouddns*](/plugins/clouddns) to enable serving zone data from GCP Cloud DNS.
 * [*sign*](/plugins/sign) that (DNSSEC) signs your zonefiles (in its most basic form).
+* [*file*](/plugins/file) various update, plug a memory leak when doing zone transfers, among other
+  things.
+* [*kubernetes*](/plugins/kubernetes) fix adding NS records for zone transfers.
```

Comment:
I don't think this is fixed. #3160 fixes  direct `NS` queries, but doesnt fix xfrs.

NS records are included in transfer in the transfer plugin PR #3223 , but thats still a WIP.

Reply:
done